index.html

```diff
@@ -1 +1,13 @@
-<meta http-equiv="Refresh" content="0; url='/speedrun-guide.html'" />
```

Comment:
have you tried redirecting to ./speedrun-guide.html?

Reply:
I tried putting them one folder deeper and doing that to test it and I'm getting a 404. It seems to not append the proper subdirectories from the previous link or something, not entirely sure. The js solution works as fast and I read that using the meta tag to redirect is bad for SEO anyway.